Lesson_5/home-work5_3/home-work5_3.py

```diff
@@ -0,0 +1,15 @@
+import os
+import re
+
+with open('../text_files_for_3d/text.txt') as file:
+    text = file.read()
+    text_s = text.lower()
```

Comment:
what does _s mean?

Reply:
I read the file, and make the text lowercase text_s = text_small

Reply:
I did changes